### ISSUE TITLE
LruCache - Lock is null

### ISSUE THREAD

**Issue by pizzi80** [omnifaces/omnifaces#803]
the internal lock is transient 
so if you restart tomcat and use a ViewScoped bean 
the internal lock is null

```
java.lang.NullPointerException: Cannot invoke "java.util.concurrent.locks.Lock.lock()" because "this.lock" is null
    at org.omnifaces.util.cache.LruCache.withLock(LruCache.java:87)
    at org.omnifaces.util.cache.LruCache.get(LruCache.java:103)
    at org.omnifaces.util.cache.LruCache.get(LruCache.java:42)
    at org.omnifaces.cdi.viewscope.ViewScopeStorageInSession.getBeanStorage(ViewScopeStorageInSession.java:83)
    at org.omnifaces.cdi.viewscope.ViewScopeStorageInSession$Proxy$_$$_WeldClientProxy.getBeanStorage(Unknown Source)
    at org.omnifaces.cdi.viewscope.ViewScopeManager.getBeanStorage(ViewScopeManager.java:192)
    at org.omnifaces.cdi.viewscope.ViewScopeManager.getBean(ViewScopeManager.java:128)
    at org.omnifaces.cdi.viewscope.ViewScopeManager$Proxy$_$$_WeldClientProxy.getBean(Unknown Source)
```

**Comment by BalusC** [comment]
Ay crap. Overlooked that part. I'm having vacations atm and can't quickly release a hotfix. You'll need to have patience until sometime after may 4.

**Comment by pizzi80** [comment]
No worries, have a nice holiday!

... in the meantime I'll use a local build of my fork ... ;)

I've have made a lot of PR(s) :stuck_out_tongue_winking_eye:

**Comment by BalusC** [comment]
Oh, this duplicates #802. I've already fixed it via that ticket and will release 4.4.1 as soon as possible when there are no unworkaroundable 4.4 bugs in remaining issues.

**Comment by BalusC** [comment]
4.4.1 has been released and will be in Maven central soon.

Thank you very much for reporting!
